site/content/docs/Advanced/Kopiaignore/_index.md

```diff
@@ -8,12 +8,12 @@ weight: 40
 
 Users may want to exclude folders and files not to be saved within the repository when creating snapshots. The benefits of omitting unnecessary files and folders are smaller and faster snapshots while saving only essential data. 
 
-Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such patterns. 
-If such a file is placed within a directory, `Kopia` omits files and folders `matching` the pattern.
+Kopia features `pattern-based` ignore rules to omit folders and files from snapshots. While scanning directories and their content, Kopia looks explicitly for files that contain such rules. 
+If such a file is placed within a directory, `Kopia` omits files and folders `matching` the rule.
```

Comment:
plural? there may be more than one rule in an ignore file, right?
I guess my confusion here is regarding terminology
What is a rule vs what is a pattern?
```suggestion
If such a file is placed within a directory, `Kopia` omits files and folders `matching` the rules.
```